getMultiStarts.m

```diff
@@ -295,7 +318,7 @@
                 % parameters.constraints.A  ,parameters.constraints.b  ,... % linear inequality constraints
                 % parameters.constraints.Aeq,parameters.constraints.beq,... % linear equality constraints
                 
-                parameters.MS.J(1, i) = -J_0;
+                parameters.MS.J(1, i) = nan;
```

Comment:
parameters.MS.J is the posterior value at the initial point. For Meigo, this doesn't make sense, since there are many initial points.